public/app/features/dashboard/state/DashboardMigrator.ts

```diff
@@ -223,6 +225,7 @@ export class DashboardMigrator {
     // schema version 10 changes
     if (oldVersion < 10 && finalTargetVersion >= 10) {
       // move aliasYAxis changes
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Why do you `// eslint-disable-next-line @typescript-eslint/no-explicit-any`? Is there any new linting rule that breaks this?